install/gcp/deployment_manager/istio-cluster.jinja

```diff
@@ -97,7 +98,31 @@ resources:
           {% endif %}
 
           {% if  properties['enableAutomaticSidecarInjection'] %}
-            kubectl apply -f install/kubernetes/istio-initializer.yaml
+            {% if properties['installIstioRelease'] == "0.5.0" %}
+              # Setup automatic istio sidecar injection for the default namespace.
+              # Note that the original 0.5.0-tagged version of this script had
+	      # a timing bug; here we pull one commit later.
+              curl -s https://raw.githubusercontent.com/istio/istio/41203341818c4dada2ea5385cfedc7859c01e957/install/kubernetes/webhook-create-signed-cert.sh -o ./install/kubernetes/webhook-create-signed-cert.sh && chmod +x ./install/kubernetes/webhook-create-signed-cert.sh
```

Comment:
Shouldn't these scripts come within the release .zip by default?  (its weird to download them out of band (not just here in this script but generally)

Istio's docs are missing calling this step [here](https://istio.io/docs/setup/kubernetes/sidecar-injection.html#installing-the-webhook).   I'll file an issue to include the script into the ```0.5.1``` release

Reply:
They should be part of istioctl at some point. Brand new code, polish in progress.

Reply:
they are part of the release now but may have missed 0.5.0 but will be in 0.5.1 and 0.6.0